### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by simonwep** [simonwep/ocular#95]
### Support guidelines

- [x] My request extends the functionality of Ocular in a meaningful way.
- [x] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Right now, due to missing `aria-label` and `label` elements the UI, especially income and expense tables, are hardly navigate-able with a screenreader.

### Solution

Add `aria-label` where applicable and missing `label`'s for input elements.

**Comment by dnunnari** [comment]
@simonwep could I help out with this one?

**Comment by simonwep** [comment]
Hey @dnunnari! Of course :) I think the first step, before even testing it with a screen-reader which will be a bit tricky since there is no one-fits-all, would be to add `aria-label` to all cells in the income/expense budget panes to label each cell with the corresponding row name and month. For example for this

![Image](https://github.com/user-attachments/assets/ffe6fcaf-3032-4d29-8c61-dd9f7a80da34)

the aria label may read "Income for dividends in februrary". Of course the other UI Elements in a `BudgetPane` (the component that renders both income and expenses) should also be labelled in an appropriate way. But I think this would be a good start.

Right now the browser reports issues that all these input elements don't have `label`'s associated with them, but I'm afraid it'll be too much for the DOM if we add a `sr-only` `label` for each which would be the better solution in most cases, but you can also try that.

**Comment by simonwep** [comment]
Actually fixed in 50ee1d684d093aecd1a323144a0d191fd7bc15e4
